doc/api/api_changes_3.3/deprecations.rst

```diff
@@ -554,3 +554,7 @@ in which they are given.  This only affects the interaction between the
 properties: the *color* property now needs to be passed first in order not to
 override the other properties.  This is consistent with e.g. `.Artist.update`,
 which did not reorder the properties passed to it.
+
+Passing multiple keys as a single comma-separated string or multiple arguments to `.ToolManager.update_keymap`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This is deprecated; pass keys as a list of strings instead.
```

Comment:
```suggestion
This is deprecated; pass keys as a single string or a list of strings instead.
```

Reply:
I thought "single string" was too confusing (as it used to be a single string before too... I guess it should be "single string if you're only passing a single key", but heh) and just documenting "list of strings" here was good enough (even though we support the single string option too).

Reply:
Ok, since this is just the deprecation notice and not the reference docstring, the imprecision is tolerable 😄 .